### PR TITLE
data: Amalks920

### PR DESCRIPTION
**Pull request by Amalks920**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5456"></a>

### CODE REVIEW

**Review by sidharthviz — APPROVED** [review]
LGTM 👍🏼

**Review by mahfuztbt — APPROVED** [review]

File: data/Amalks920.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		},
+
+		{
+			"name": "Follow me on GitHub",
+			"url": "https://github.com/Amalks920",
+			"icon": "FaGithub"
+		},
+		{
+			"name": "Follow me on Hashnode",
+			"url": "https://hashnode.com/",
```
Comment (kumarsonsoff3) [inline data/Amalks920.json:18]: You might want to add your Hashnode Profile URL here!? 🛠

Comment (vedprakashnautiyal) [inline data/Amalks920.json:18]: The url you provided redirects to hashnode feed, you might want to redirect it to your blog page.......

**Review by vedprakashnautiyal — CHANGES_REQUESTED** [review]

**Review by SurajPratap10 — APPROVED** [review]

File: data/Amalks920.json (hunk 1 above)
Comment (SaraJaoude) [reply data/Amalks920.json:18]: > The url you provided redirects to hashnode feed, you might want to redirect it to your blog page.......

@vedprakashnautiyal please avoid duplicating your suggestion as this causes confusion to the author of the PR

**Pull request merged**